src/opentime/rationalTime.cpp

```diff
@@ -94,6 +94,69 @@ is_dropframe_rate(double rate)
     return std::find(b, e, rate) != e;
 }
 
+static bool
+parseFloat(char const* pCurr, char const* pEnd, bool allow_negative, double* result)
+{
+    if (pCurr >= pEnd || !pCurr) {
+        *result = 0.0;
+        return false;
+    }
+
+    double ret = 0.0;
+    double sign = 1.0;
+
+    if (*pCurr == '+')
+        ++pCurr;
+    else if (*pCurr == '-') {
+        if (!allow_negative) {
+            *result = 0.f;
```

Comment:
Some places in this code use `0.f` and others `0.0` is there a meaningful difference?

Reply:
0.f is a floating point constant, 0.0 is a double. It should be 0.0 here, although it won't affect codegen.